README.md

```diff
@@ -107,24 +131,29 @@ cd webvm
 	location /disk-images/ {
         root .;
         autoindex on;
+		gzip off;
```

Comment:
Gzip is only enabled on some mimetypes, which should _not_ cover the disk image. I think the issue you had is connected to something else and this fix is spurious.

The while disk-images section is not required to begin with, the image can be placed anywhere on the server